### PR TITLE
Services, Utility: fix revision counter calculation

### PR DESCRIPTION
**Pull request by aarani**

### CODE REVIEW

File: NOnion/Utility/HiddenServicesUtility.fs [hunk 1]
```diff
@@ -34,6 +34,22 @@ module HiddenServicesUtility =
 
         unixNow - timeElapsedSinceStartOfRun |> FromUnixTimestamp
 
+    let GetStartTimeOfPreviousSRVProtocolRun
+        (now: DateTime)
```
Comment (knocte) [inline NOnion/Utility/HiddenServicesUtility.fs:38]: why is this param called now? I think it needs a better name

Comment (knocte) [reply NOnion/Utility/HiddenServicesUtility.fs:38]: @aarani

Comment (aarani) [reply NOnion/Utility/HiddenServicesUtility.fs:38]: There's already a function just above this that has the same name, I prefer to be consistant.

Comment (knocte) [reply NOnion/Utility/HiddenServicesUtility.fs:38]: sure, be consistent in that commit, but then rename it in a next commit in both functions

File: NOnion/Utility/HiddenServicesUtility.fs [hunk 2]
```diff
@@ -6,9 +6,10 @@ open NOnion
 open DateTimeUtils
 
 module HiddenServicesUtility =
-    let GetTimePeriod (now: DateTime) (hsDirInterval: int) =
+    let GetTimePeriod (lastConsensusValidAfter: DateTime) (hsDirInterval: int) =
```
Comment (knocte) [inline NOnion/Utility/HiddenServicesUtility.fs:9]: I said the rename should go in a different commit, man

Comment (aarani) [reply NOnion/Utility/HiddenServicesUtility.fs]: Yeah, I know, fixed it as soon as I noticed.

File: NOnion/Utility/HiddenServicesUtility.fs [hunk 3]
```diff
@@ -6,9 +6,10 @@ open NOnion
 open DateTimeUtils
 
 module HiddenServicesUtility =
-    let GetTimePeriod (now: DateTime) (hsDirInterval: int) =
-        let nowInMinutes =
-            let validAfterSinceEpoch = now |> GetTimeSpanSinceEpoch
+    let GetTimePeriod (liveConsensusValidAfter: DateTime) (hsDirInterval: int) =
```
Comment (knocte) [inline NOnion/Utility/HiddenServicesUtility.fs:9]: now that you're renaming stuff, let's remove the hs acronym here

**Review by knocte — APPROVED** [review]
